test/UBI.js

```diff
@@ -325,6 +325,32 @@ contract('UBI', ([_, owner, recipient, attacker, systemOwner]) => { // eslint-di
       bal.should.bignumber.satisfy(() => near(bal, goal, startingIssuance));
     });
 
+    it('should show no payable ubi if timeout is shorter than period', async () => {
```

Comment:
Can this be moved into its own describe block, and follows the pattern of deploying a hub in the before each like the rest of this file?  

Would suggest to run a version of all the same tests we use on other hub parameters (and probably mostly use the same code):

- should return that it is stopped when it has been manually stopped
- should not payout ubi if manually stopped
- should show no payable ubi if expired
- should return that it is stopped when it has expired
- should not payout ubi if expired

`expired` here meaning `timeout` has passed, and `manually stopped` meaning the stop method has been called